Apps/APN/ios/MyAppPushNotificationsHandler.swift

```diff
@@ -16,14 +15,9 @@ public class MyAppPushNotificationsHandler : NSObject {
 
   @objc(setupCustomerIOClickHandling)
   public func setupCustomerIOClickHandling() {
-    // This line of code is required in order for the Customer.io SDK to handle push notification click events.
-    // We are working on removing this requirement in a future release.
-    // Remember to modify the siteId and apiKey with your own values.
-    CustomerIO.initialize(siteId: Env.siteId, apiKey: Env.apiKey, region: .US) { config in
-      config.autoTrackDeviceAttributes = true
-      config.logLevel = .debug
-    }
-    MessagingPushAPN.initialize(configOptions: nil)
+    // Initialize MessagingPushFCM module to
+    // automatically handle your app’s push notifications that originate from Customer.io
+    MessagingPushAPN.initialize(withConfig: MessagingPushConfigBuilder().build())
```

Comment:
I think we have discussed this on FCM before. We only need this in NSE and not in app I think? 🤔

Reply:
We are not initializing the SDK but `MessagingPushAPN` module but I could be mistaken. To confirm are you referring to [this comment](https://github.com/customerio/customerio-reactnative/pull/303#discussion_r1721219301) ?